test/ng/directive/inputSpec.js

```diff
@@ -3164,6 +3240,34 @@ describe('input', function() {
           expect(scope.form.alias.$error.max).toBeFalsy();
         });
 
+        it('should not assume a max val of 100 if the max attribute interpolates to a non-number', function() {
```

Comment:
I don't see how this test illustrates this description. (But I am not sure if it is the description or the test or the implementation that needs to change :smiley:)